File: src/main/java/me/arasple/mc/trmenu/action/acts/ActionActionbar.java
package me.arasple.mc.trmenu.action.acts;

import io.izzel.taboolib.module.locale.TLocale;
import me.arasple.mc.trmenu.action.base.AbstractAction;
import me.arasple.mc.trmenu.utils.Vars;
import org.bukkit.entity.Player;

/**
 * @author Arasple
 * @date 2019/12/28 18:49
 */
public class ActionActionbar extends AbstractAction {

    @Override
    public String getName() {
        return "actionbar";
    }

    @Override
    public void onExecute(Player player) {
        TLocale.Display.sendActionBar(player, Vars.replace(player, getContent()));
    }

}

File: src/main/kotlin/me/arasple/mc/trmenu/modules/action/impl/ActionActionbar.kt
package me.arasple.mc.trmenu.modules.action.impl

import io.izzel.taboolib.module.locale.TLocale
import me.arasple.mc.trmenu.modules.action.base.Action
import org.bukkit.entity.Player

/**
 * @author Arasple
 * @date 2020/3/8 20:39
 */
class ActionActionbar : Action("action(bar)?(s)?") {

	override fun onExecute(player: Player) = TLocale.Display.sendActionBar(player, getContent(player))

}